stdlib/http/src/main/java/org/ballerinalang/net/http/websocket/WebSocketUtil.java

```diff
@@ -33,29 +33,38 @@
 import org.ballerinalang.jvm.values.connector.NonBlockingCallback;
 import org.ballerinalang.net.http.HttpConstants;
 import org.ballerinalang.net.http.HttpErrorType;
+import org.ballerinalang.net.http.websocket.client.RetryContext;
+import org.ballerinalang.net.http.websocket.client.WebSocketClientConnectorListener;
+import org.ballerinalang.net.http.websocket.client.WebSocketClientHandshakeListener;
 import org.ballerinalang.net.http.websocket.observability.WebSocketObservabilityUtil;
 import org.ballerinalang.net.http.websocket.server.WebSocketConnectionInfo;
 import org.ballerinalang.net.http.websocket.server.WebSocketConnectionManager;
 import org.ballerinalang.net.http.websocket.server.WebSocketServerService;
 import org.ballerinalang.stdlib.io.utils.IOConstants;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.wso2.transport.http.netty.contract.websocket.ClientHandshakeFuture;
+import org.wso2.transport.http.netty.contract.websocket.WebSocketClientConnector;
 import org.wso2.transport.http.netty.contract.websocket.WebSocketConnection;
 
 import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
 import java.util.Locale;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
 
 import javax.net.ssl.SSLException;
 
-import static org.ballerinalang.net.http.websocket.WebSocketConstants.ErrorCode;
-import static org.ballerinalang.stdlib.io.utils.IOConstants.IO_PACKAGE_ID;
-
 /**
  * Utility class for WebSocket.
  */
 public class WebSocketUtil {
 
     private static final Logger logger = LoggerFactory.getLogger(WebSocketUtil.class);
+    private static final String STATEMENT_FOR_RECONNECT = "Maximum retry attempts but couldn't connect " +
```

Comment:
As this a statement and is used only used in one place, shall we not define it as a constant?